src/vm/method.cpp

```diff
@@ -2149,12 +2146,29 @@ PCODE MethodDesc::TryGetMultiCallableAddrOfCode(CORINFO_ACCESS_FLAGS accessFlags
 
     // We create stable entrypoints for these upfront
     if (IsWrapperStub() || IsEnCAddedMethod())
+    {
+        if (IsZapped() && !HasStableEntryPoint())
```

Comment:
This condition should not be needed. We should be able to just call GetOrCreatePrecode() all the time. (It would be tiny bit more efficient.)

Reply:
(It can be just assert.)